libclc/generic/include/clc/async/async_work_group_copy.h

```diff
@@ -9,15 +9,31 @@
 #define __CLC_DST_ADDR_SPACE local
 #define __CLC_SRC_ADDR_SPACE global
 #define __CLC_BODY <clc/async/async_work_group_copy.inc>
-#include <clc/async/gentype.inc>
+#include <clc/integer/gentype.inc>
+#undef __CLC_DST_ADDR_SPACE
```

Comment:
We probably don't need to `undef` and re-`define` here, when `local/global` isn't changing. Obviously we do when we swap them further down.

Reply:
done, thanks for the suggestion